scripts/populate_sample_data.py

```diff
@@ -63,9 +68,9 @@
 CLASSROOM_URL_FRAGMENT: Final = 'math'
 
 
-class GenerateSampleData:
-    """Contains functions that populate sample data by sending requests to the
-    development server.
+class DebugInitializer:
```

Comment:
This still implies that the sample data is for debugging only. Also, the file is currently only for the contributor dashboard, so can we reflect that in the name? Maybe `populate_sample_contributor_data` and `SampleDataInitializer`?

Reply:
Done.